holidayextras/plugin-elephant
# WEB-9393 Amend Payment Methods to detect when the card has expired

#### What does this PR do? (please provide any background)
https://hxshortbreaks.atlassian.net/browse/WEB-9393

The Works is going to select the most recently created active Payment Method as a default form of payment.

This is the version bump for the plugin

#### What tests does this PR have?
nada - bumping versions.

#### How can this be tested?
Can use the branch web9393_use_payment_methods on the works, and see that amends and cancels are still working.

#### Any tech debt?
no

#### Screenshots / Screencast

#### What gif best describes how you feel about this work?
![]()


---

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

---
